config.rb.sample

```diff
@@ -42,6 +42,9 @@
 #   export DOCKER_HOST='tcp://127.0.0.1:2375'
 #$expose_docker_tcp=2375
 
+# Enable port forwarding from guest to host machine, syntax is: { guestPort1 => hostPort1, ... }, auto correction is enabled by default.
```

Comment:
It makes a little more sense to me if these are defined as host => guest. Is there any prior art for this?